webapp/src/container.tsx

```diff
@@ -45,6 +45,12 @@ function startTour(parent: IProjectView) {
     parent.showOnboarding();
 }
 
+function openKeyboardNavFn(parent: IProjectView) {
+    return function() {
+        parent.toggleShortcutDialog();
+    }
+}
+
```

Comment:
This can be simplified to

```suggestion
function openKeyboardNavFn(parent: IProjectView) {
        parent.toggleShortcutDialog();
}

```

if the caller is changed to `{() => openKeyboardNavFn(parent)}`